modules/ROOT/pages/dashboard-custom-config-graph.adoc

```diff
@@ -211,15 +215,15 @@ image::graph-advanced-threshold.png[Threshold Settings for a Graph]
 2+| Threshold Settings
 | `gt` or `lt` | Indicates whether the limit applies to data that is greater than (`gt`) or less than (`lt`) the specified threshold value.
 | Threshold value | A numeric value for the threshold.
-| Threshold type | Identifier for the threshold. One of Custom, Critical, Warning, or OK.
+| Threshold type | Identifier for the threshold (Custom, Critical, Warning, or OK).
 | Fill | Adds a filler color to the graph that corresponds to the specified threshold setting.
 | Line |  Adds a line to the graph that corresponds to the specified threshold setting.
-| Fill Color  | Similar to Fill settings, but for custom thresholds only.
-| Line Color | Similar to Line settings, but for custom thresholds only.
+| Fill Color  | Similar to *Fill* settings, but for custom thresholds only.
+| Line Color | Similar to *Line* settings, but for custom thresholds only.
 |===
 
 === Time Range Shift (Advanced Setting)
-In the Advanced configuration tab, you can set overrides any time intervals or ranges that end users select for the custom dashboard. Such selections are global and apply to all charts in the dashboard. Overrides are useful when the selection does not make sense for a particular graph.
+In the *Advanced* configuration tab, you can set overrides any time intervals or ranges that end users select for the custom dashboard. Such selections are global and apply to all charts in the dashboard. Overrides are useful when the selection does not make sense for a particular graph.
```

Comment:
set overrides on any time intervals? for any time intervals? Is a chart different from a graph?